zeppelin-web/src/components/websocket/websocket-event.factory.js

```diff
@@ -172,7 +172,7 @@ function WebsocketEventFactory($rootScope, $websocket, $location, baseUrlSrv, sa
         closeByBackdrop: false,
         closeByKeyboard: false,
         title: 'Details',
-        message: _.escape(data.info.toString()),
+        message: _.escape(data.info.toString()).replace(/\n/g, '<br>'),
```

Comment:
I added newline processing to the error popup message. It could help users better understand the message.

Reply:
I don't know what message is being displayed here. If it is the `CorruptedNoteException`, then I think it is unnecessary.
See comment below on the multi-line error message.